test/ssa.spec.js

```diff
@@ -0,0 +1,144 @@
+/* eslint-env mocha */
```

Comment:
tidy up file, remove unnecessary tests, add new ones. Include outputs folder to test